soc/realtek/ameba/soc.yml

```diff
@@ -0,0 +1,6 @@
+family:
+- name: realtek_ameba
+  series:
+  - name: amebadplus
+    socs:
+    - name: amebadplus
```

Comment:
Is this really true, you have a line of products called `amebadplus` and one specific model on that line is called `amebadplus`? Series is not required, if it is not needed then drop the series